apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -506,6 +506,20 @@
   </script>
 
 </nav>
+
+<% flash(['notice', 'success', 'warn', 'error']).forEach(({ type, message }) => { %>
+  <% const color = type === 'warn' ? 'warning' : type === 'error' ? 'danger' : type === 'notice' ? 'info' : 'success'; %>
+  <div class="alert alert-<%= color %> border-left-0 border-right-0 rounded-0 mb-0" role="alert">
+    <div class="container">
+      <div class="row">
+        <div class="col">
+          <%= message %>
+        </div>
+      </div>
+    </div>
+  </div>
+<% }); %>
```

Comment:
Open question: is this the right place? On the instructor assessment groups page (the only page using flash-like things at the moment), errors were shown below the nav tabs, but before I saw that, I added it above the tabs.

In the following screenshot, "Nathan testing ..." is the old location, and the other error is the new one.

<img width="332" alt="Screenshot 2023-06-05 at 16 57 50" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/15f524b3-5897-405e-8e6c-844f75748790">

Reply:
I'm more in favour of the old location. It looks closer to the actual page functionality.

Reply:
I think I agree. However, the old location has the downside that we don't know if a page is normal-width or full-width when we render the navbar, so we can't make the width of the flash messages match that of the page itself.

Reply:
Can't you just put this directly into the page, using the `container` or `container-fluid` that's already there? This will then automatically make it the right width.

Reply:
I really want to make this part of our nav so we don't have to repeat this snippet on every page.

GitHub renders these as a full-width thing all the time *and* renders blow the nav tabs. I think this is a good option?

<img width="1534" alt="Screenshot 2023-06-06 at 18 23 34" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/b8237e45-01c3-41c0-82f9-0baab814ab7e">

Reply:
I'm happy with that option.

Reply:
Agreed.

Reply:
Moved below the tabs in 13577ec. I enabled dismissal, which required a small amount of additional CSS/JS.

Reply:
Actually, I was able to achieve the desired behavior _without_ any extra CSS/JS! GitHub isn't showing the commit yet; I'll link to it once it's there.